core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/preference/editorproperties/EditorProperties.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.jseditor.client.preference.editorproperties;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * The interface contains editor's constants.
+ *
+ * @author Roman Nikitenko
+ */
+public enum EditorProperties {
+
+    TAB_SIZE("tabSize"),
```

Comment:
Can you add also property for showing white spaces?

Correspondent property name is `showWhitespaces`.